src/lib/configure-security.ts

```diff
@@ -21,23 +21,23 @@ export default function configureSecurity(apiDefinition: OASDocument, values: Au
 
   if (isRef(security)) {
     return undefined;
+  } else if (!values[scheme]) {
+    // If we don't have any data for this auth scheme then we shouldn't add it.
+    return false;
   }
 
   if (security.type === 'http') {
     if (security.scheme === 'basic') {
-      // Return with no header if user and password are blank
-      if (!values[scheme]) return false;
-
       const auth = values[scheme];
       if (typeof auth !== 'object') return false;
       if (!auth.user && !auth.pass) return false;
 
-      let user = auth.user;
+      let user = auth.user ?? null;
```

Comment:
Also found this problem while testing. We always set `user` and `pass` to `null` or an empty string in ReadMe, but if for whatever reason it wasn't specified we'd crash here while trying to do `undefined.length === 0`.